modules/client-configuration/pages/clients-sle.adoc

```diff
@@ -78,6 +78,18 @@ The channels you need for this procedure are:
 
 |===
 
+[NOTE]
+====
+To find channel names of older products, at the command prompt on the {productname} Server, as root, use the [command]``mgr-sync`` command:
+
+----
+mgr-sync list --help
+----
+
+Then specify the argument you are interested in.
+For example, [literal]``-c channels``.
```

Comment:
`-c` is at the wrong place (funny - it works anyway). But we write better:

```
mgr-sync list channels [-c]
```
The option comes after the subcommand. "-c" is here "compact".

Reply:
ok, thanks `;-)`